content/topics/reopening-pull-request/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+_db_id: 
+content_type: topic
+ready: true
+title: Reopening a Pull request
+---
+
+Handling requested changes on pull requests
+
+Pull requests are an excellent approach to communicating proposed code changes for review and receiving feedback from staff and other learners. This usually leads to dialogues and suggestions in the comments, which may lead to more code changes.
+
+While comments and feedback are useful for enhancing code quality, they can easily be lost. To ensure that data isn't lost and a consistent system of record is created, it is worth noting that you are expected to REOPEN a closed Pull request if changes are requested. 
```

Comment:
This sentence feels off, consider revising it.

Maybe:

`To ensure that data isn't lost and a consistent record of the feedback is created, you must REOPEN a closed Pull request once changes have been pushed. `

Feel free to adjust further